src/backend/core/models.py

```diff
@@ -452,6 +454,41 @@ def get_abilities(self, user):
         }
 
 
+class Invitation(BaseModel):
+    """User invitation to teams."""
+
+    email = models.EmailField(_("email address"), null=False, blank=False)
+    team = models.ForeignKey(
+        Team,
+        on_delete=models.CASCADE,
+        related_name="invitations",
+    )
+    role = models.CharField(
+        max_length=20, choices=RoleChoices.choices, default=RoleChoices.MEMBER
+    )
+    issuer = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="invitations",
+    )
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["email", "team"], name="email_and_team_unique_together"
+            )
+        ]
```

Comment:
I added uniqueness together to prevent duplication of invitation to the same team. 

An email can be invited to several teams though, so we'll have to decide if we send an invitation mail each time or not.

Reply:
There should be a diff for the migration file as well?

Reply:
You're absolutely right, I had forgotten to do it again